### PR TITLE
About page

### LINKED ISSUE #53: Add About page with app description
About page could contain instructions on how to use the app, explanations on colour coding if we decide to implement it, and names of its creators

### PR DESCRIPTION
**Pull request by N-Zubko**
## Description

🚧 Work in Progress 🚧 but your opinion needed

I have created the About page, which will contain basic info about the app, instructions on how to use it, names of our mentors and our names.

The page is not completed, but I wanted you to have a look at it, check the page content and help correct phrases whenever it is needed, share thoughts on the layout and the color scheme.

To be added later:
1. About button (with a question mark instead of words) to the Home screen
2. About button to the top bar displayed in the List and Add Item pages
3. Links to our Github and LinkedIn profiles in the Creators section of About
4. Return button to go back from the About page

## Related Issue

Closes #53 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
Screen Shot of About page for Shop-Ade app

## Testing Steps / QA Criteria

Check out `NZ-about-page` and run `npm start`. To access the page add `/about` after localhost

### CODE REVIEW

**Comment by N-Zubko** [comment]
We could use the content of this page in the READ.ME file

**Review by Trevor-Rezac — COMMENTED** [review]
Overall I think this is a great start! I did notice a few typos in some of the text. Also, this is just my opinion but I think the about page looks too wordy. Today it's hard to keep people's attention online and a large wall of text may be off putting. I have some ideas to organize the content in way that makes it easier to skim and find what you need. Something like... 

Getting Started (this would be a header)
- Create a new list by clicking 'Create List'
OR
- Join an already existing list by entering the list name and clicking 'Join List'

Adding Items
- Click the 'Add Item' button
- Select the appropriate purchase timeframe for the item

Purchase Timeframes
- Soon (item is likely to be bought again within 7 days)
- Kind of soon (Item is likely to be bought again within 8-14 days)
- Not so soon (item is likely to be bought again within 15-30 days)
- Inactive (items not marked as purchased for over 30 days will appear in the inactive section)

Editing/Deleting Items
- You may edit an item on your list by clicking the 'Edit' button
- Retype the name of the item and click 'Submit' or hit enter
- To delete an item from your list, click the 'X' button

This was just really quick on the fly but I think this would make the about section more readable and, in the event I am just looking for a specific piece of information, I should be able to quickly skim and find what I need. 

Thoughts?

**Comment by michellerenehey** [comment]
This is SUCH a great start! I love the buckets: **What is Shop-Ade**, **How to use Shop-Ade**, and **Creators**. I left a few specific comments for some of the text in the first and last buckets. 

Regarding the middle bucket (how to use Shop-Ade) - I don't think we need to be so specific. Our app is pretty intuitive regarding how to navigate it (with buttons like 'Add Item' and 'Log Out' and 'Create List') - I just don't think we need to go into so much detail. 

So! With that said! My opinion would be to keep this 'How to use Shop-Ade' section just for the bits and pieces that a user might not understand! Truth-be-told it took me a _WHILE_ to understand how the app works. Here is my suggested edit: 

"Getting started with Shop-Ade is easy! On your first visit, simply create a new list and start adding the items that you need to purchase. Include the frequency at which you typically buy them to help the app get smarter (do you buy toilet paper on the weekly? lobster rolls every month?). 

Once you have a few items in your list, the app will help keep you organized by managing your purchase frequency. Did you mark 'lobster rolls' as a 'not so soon' purchase, but then you bought them today? We will help you better understand that maybe, in fact, you tend to purchase them weekly (or daily?!) instead of monthly. Next time you look at your list, you'll see the adjustments to your typical purchase timelines. We're here for your organizing needs. 

You can interact with your list items to help the app get _even_ smarter: mark the items as purchased (if you mark an item as purchased within the timeframe you selected, it will stay in the specified timeframe bucket; if you purchase an item sooner or later than you thought, it will move timeframe buckets accordingly); edit the item as you wish; and delete it when/if you're finished (actually, we don't need those monthly lobster rolls...). 

The more you use the app the more it learns your behaviors! Have fun!"

File: src/views/About.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			<h1>About</h1>
+			<article className="about-article">
+				<h2 className="article-title">What is Shop-Ade</h2>
+				<p>
+					<b>Shop-Ade</b> is a smart shopping list that learns your buying
+					habits and helps you remember what you need to buy on your next trip
+					to the store. Each time you buy the item, you mark it as purchased in
+					the list. Over time, the app comes to understand the intervals at
+					which you buy different items. If an item is likely to be due to be
+					bought soon, it rises to the top of the shopping list.
```
Comment (lindseyindev) [inline src/views/About.jsx:15]: I think this is an excellent description

File: src/views/About.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+						the item name. You will see a confirmation dialog, if you select OK
+						in the dialog window the item will be immediately deleted.
+					</li>
+					<li>
+						If you bought an item - mark it as purchased in the list by putting
+						a checkmark. The item will uncheck itself after 24 hours. The app
+						will use the purchase information to predict next purchase and will
+						move the item to one of the four time frames.
+					</li>
+					<li>
```
Comment (lindseyindev) [inline src/views/About.jsx:53]: I think the content here is good -- potentially not needed but if you do want to keep it I would recommend nesting the content under expandable sections so that you can skip over the sections you don't need info on quickly

like 

<details><summary>Adding to your list   </summary>If you want to add an item to your list - click on Add Item button,
						enter the item name, select the time frame when you want to buy this
						item out of three options - "soon", which means within 7 days, "kind
						of soon" which means within 8-14 days, "not so soon", which means
						within 15-30 days.</details>
<details><summary>Editing the list   </summary>If made a mistake when entering the item name - you can re-enter the
						name by selecting Submit button. Click Submit when you finish
						correcting the item name. The name will be immediately updated.</details>
<details><summary>Deleting items from the list   </summary> 
						If you want to delete an item from your list - click X button near
						the item name. You will see a confirmation dialog, if you select OK
						in the dialog window the item will be immediately deleted.</details>
<details><summary> Updating purchases  </summary>If you bought an item - mark it as purchased in the list by putting
						a checkmark. The item will uncheck itself after 24 hours. The app
						will use the purchase information to predict next purchase and will
						move the item to one of the four time frames.</details>

Comment (lindseyindev) [reply src/views/About.jsx:53]: This might be a personal preference but if it interests you to have these expandable sections  the markup for this is using the details and summary html elements like this:

 `<details><summary>{Brief Description}</summary>{Long Description} </details>`

Comment (lindseyindev) [reply src/views/About.jsx:53]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

Comment (N-Zubko) [reply src/views/About.jsx:53]: So many HTML native elements I am to discover! Thank you @lindseyindev 
I really like the idea of expandable sections

Comment (Trevor-Rezac) [reply src/views/About.jsx:53]: I actually really like this idea too. I think it would be good to have a brief description of the app, but then have these expandable sections for more detailed instructions. I think it gives some more life to the app!

**Review by mira-kine — APPROVED** [review]
Clean description and directions on using the app! Never seen the use of <details> and <summary> in the html, makes it easy to read.
